scripts/start_manager.sh

```diff
@@ -44,11 +41,6 @@ if [ -n "$manager_pid" ]; then
 fi
 
 cd "${MANAGER_DIR}" && ~/.cargo/bin/cargo build $RUST_BUILD_PROFILE $PG_EMBED
-cd "${MANAGER_DIR}" && ~/.cargo/bin/cargo run --bin pipeline-manager $RUST_BUILD_PROFILE $PG_EMBED -- \
+cd "${ROOT_DIR}" && ~/.cargo/bin/cargo run --bin pipeline-manager $RUST_BUILD_PROFILE $PG_EMBED -- \
     --bind-address="${BIND_ADDRESS}" \
```

Comment:
you shouldn't need any of the rest either. This should do.

```
cargo run --bin=pipeline-manager --features pg-embed
```

Reply:
The rest is there so this script can be invoked with non-default settings. Not sure if anyone needs that...